packages/itwin/ec3-widget/src/components/EC3/EC3Config.ts

```diff
@@ -61,6 +62,11 @@ export interface EC3ConfigCommonProps {
    * A custom implementation of EC3ConfigurationClient. If provided, carbonCalculationBasePath is ignored.
    */
   ec3ConfigurationsClient?: IEC3ConfigurationsClient;
+
+  /**
+   * A custom implementation of useEC3WidgetLocalizationResult. If provided, corresponding default strings will ignored.
```

Comment:
... will **be** ignored.

Reply:
removed this prop, as it is no longer required